src/main/java/eu/europa/ec/fisheries/uvms/exchange/model/mapper/ExchangeModuleRequestMapper.java

```diff
@@ -610,8 +606,6 @@ private static void populateBaseProperties(ExchangeBaseRequest request, String f
         request.setSenderOrReceiver(senderReceiver);
         request.setOnValue(onValue);
         request.setRegisteredClassName(registeredClassName);
-        request.setTo(to);
```

Comment:
Again this method is only used to send to exchange, you should not remove these fields!